README.rst

```diff
@@ -149,6 +149,28 @@ Advanced Usage
 
 See below for more options.
 
+Order Your Grouped Results with order_by and limit
+''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Alpha support for ordering your results and limiting what is displayed is now supported in the Python SDK.
```

Comment:
"displayed" is a little awkward here, maybe "returned" 

what do you think?

Reply:
Displayed is deliberate to make sure people know the entire set of data is being read. Knowing that, do you think I should change it to "returned"?

Reply:
Yeah, I figured that was why. The thing is, it is also what is returned. Like, yes, we are getting more results, but only return a portion of them. 

We could possibly add a note somewhere letting people we are still reading the entire set of data. Not sure if that belongs here or in the API ref section that will be written for this with all the details.

Reply:
Given that it is listed elsewhere I'm happy to change this to "returned" in this blurb.